.github/workflows/ci.yml

```diff
@@ -84,3 +84,29 @@ jobs:
         run: ${{ matrix.test }}
         env:
           TARGET: ${{ matrix.target}}
+
+  push:
+
+    needs: test
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+
+      - name: Download Docker image artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: article-page-prod
+          path: images
+
+      - name: Load Docker image
+        run: docker load -i images/article-page-prod.tar
+
+      - name: Retag and push Docker image
```

Comment:
Though at the moment we won't retag here yet. This is envisioned as a different workflow triggered only on tagging, as it should not rebuild the image but only tag an existing one.

Reply:
I think 'retag' is still a fair description of what is going on here. LGTM.

Will be interesting to see how our various products adopt semver.